draft-tigress-general-requirements.md

```diff
@@ -109,83 +88,75 @@ General terms:
 - Provisioning Partner - an entity which facilitates Credential Information lifecycle on a device. Lifecycle may include provisioning of credential, credential termination, credential update.
 - Sender (device) - a device initiating a transfer of Provisioning Information to a Receiver that can provision this credential.
 - Receiver (device) - a device that receives Provisioning Information and uses it to provision a new credential.
-- Intermediary (server) - an intermediary server that provides a standardized and platform-independent way of transferring provisioning information between Sender and Receiver devices.
+- Intermediary (server) - an optionl intermediary server that provides a standardized and platform-independent way of transferring provisioning information between Sender and Receiver devices.
```

Comment:
it might be helpful to describe this as providing a store and forward service

Reply:
updated the description